CMakeLists.txt

```diff
@@ -580,9 +580,9 @@ if(EXISTS ${CMAKE_BINARY_DIR}/zephyr_modules.txt)
   file(STRINGS ${CMAKE_BINARY_DIR}/zephyr_modules.txt ZEPHYR_MODULES_TXT)
 
   foreach(module ${ZEPHYR_MODULES_TXT})
-    string(REGEX REPLACE "(.*):.*" "\\1" module_name ${module})
-    string(REGEX REPLACE ".*:(.*)" "\\1" module_path ${module})
-    message("Including module: ${module_name}")
+    string(REGEX REPLACE "\"(.*)\":\".*\"" "\\1" module_name ${module})
+    string(REGEX REPLACE "\".*\":\"(.*)\"" "\\1" module_path ${module})
+    message("Including module: ${module_name} in path: ${module_path}")
```

Comment:
Do you mind adding a comment about what those regex's are trying to accomplish.

Reply:
done